apps/frontend/app/admin/course/[courseId]/assignment/[assignmentId]/(overall)/@tabs/assessment/user/[userId]/problem/[problemId]/_components/SubmissionTestcase.tsx

```diff
@@ -77,27 +78,35 @@ export function SubmissionTestcase() {
 
   return (
     <div>
-      <h2 className="text-lg font-bold">Testcase</h2>
-      <Table className="**:text-center **:text-sm **:hover:bg-transparent [&_td]:p-2 [&_tr]:border-slate-600">
-        <TableHeader className="**:text-slate-100">
+      <h2 className="mb-5 text-xl font-semibold">Testcase</h2>
+      <Table className="**:border-separate **:border-spacing-0 **:text-center **:font-normal [&_td]:p-0 [&_td]:text-xs">
+        <TableHeader className="**:text-slate-100 **:text-sm **:h-[33px] **:bg-[#282D3D]">
           <TableRow>
-            <TableHead>#</TableHead>
+            <TableHead className="rounded-l-sm pl-3">#</TableHead>
             <TableHead>Result</TableHead>
             <TableHead>Runtime</TableHead>
             <TableHead>Memory</TableHead>
-            <TableHead>Score Weight</TableHead>
+            <TableHead className="rounded-r-sm pr-6">ScoreWeight</TableHead>
           </TableRow>
         </TableHeader>
+        <TableBody>
+          <TableRow>
+            <TableCell className="h-1 border-none" colSpan={5} />
+          </TableRow>
+        </TableBody>
```

Comment:
이 코드의 정체가 궁금해요! 단순히 TableBody랑 간격을 주기 위함이라면 이렇게 해도 될 것 같아요!
```suggestion
        <TableHeader className="mb-1 ...">
```